filebeat/input/file_other.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 type FileStateOS struct {
-	Inode  uint64 `json:"inode,omitempty"`
-	Device uint64 `json:"device,omitempty"`
+	Inode  uint64 `json:"inode,"`
```

Comment:
I like the idea, but thinking if that should be a generic field for all states as a registry file can only contain states for one type. What would we use the field for?